octodns_netbox/__init__.py

```diff
@@ -157,7 +157,9 @@ def _populate_PTRv4(self, zone, lenient):
 
         for ipam_record in self.ipam_records(parent=parent, family=4):
             ip_address = ip_interface(ipam_record['address']).ip
-            description = ipam_record['description']
+            dns_name = ipam_record['dns_name']
```

Comment:
The `dns_name` field is provided to hold a single name only, and employs validation to enforce this. (see: https://github.com/netbox-community/netbox/issues/4197)
At this moment, we use the `description` field as a comma-separated list of hostnames (FQDNs) so this transition will cause degradation in some use cases.

I agree that using the `dns_name` field is straightforward way, so how about creating new parameters other than `url` and `token`, and giving the user a choice of which field to use? I believe the default behavior should be the same as before for compatibility.